src/graphapi/graphapi.cpp

```diff
@@ -73,6 +74,9 @@ miopenBackendCreateDescriptor(miopenBackendDescriptorType_t descriptorType,
         case MIOPEN_BACKEND_OPERATION_RNG_DESCRIPTOR:
             outputDesciptor = new miopen::graphapi::BackendOperationRngDescriptor(); break;
 
+        case MIOPEN_BACKEND_OPERATIONGRAPH_DESCRIPTOR:
+            outputDesciptor = new miopen::graphapi::BackendOperationGraphDescriptor(); break;
```

Comment:
is it a typo? "desciptor, looks like "r" is missing

Reply:
Everywhere)

Reply:
Yes, it is a typo =)